sys/docker_build_alpine_image

```diff
@@ -0,0 +1,328 @@
+#!/usr/bin/env bash
```

Comment:
use /bin/sh, no bashisms allowed in r2land

Reply:
add .sh to the filename

Reply:
Changing to /bin/sh will likely mean a rewrite. I had to make some small changes but shellcheck now complains about lack of array support. It still runs but this could be because /bin/sh is a symlink to /bin/bash on my box. I thought that would cause bash to run in a more /bin/sh manner but I'm not sure how consistent that is across platforms/OS.